packages/@vue/cli-service/lib/Service.js

```diff
@@ -436,3 +436,14 @@ function cloneRuleNames (to, from) {
     }
   })
 }
+
+/**
+ * @typedef {import('../types/index').ConfigFunction} ConfigFunction
```

Comment:
`ConfigFunction` is not exported from '../types/index' now.  

export `UserConfig` , then ` @typedef {import('../types/index').UserConfig} UserConfig` ? Or use  `@type {import('../types/index').defineConfig}` directly?

Reply:
Yeah… Using `@type {import('../types/index').defineConfig}` should suffice. Thanks!